### PR TITLE
Add feature flag for tmglib

### LINKED ISSUE #9: Is libtmglib necessary?
In my particular use case, I'm (ab)using `netlib-src` to link to a system BLAS/LAPACK-like library, which is unlikely to have to `libtmglib.so`.  Given that it's mostly used for testing, perhaps there should be a way to at least disable linkage to it.

### PR DESCRIPTION
**Pull request by Rufflewind**
Fixes #9.

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -38,7 +39,9 @@ fn main() {
     }
     if lapacke {
         println!("cargo:rustc-link-lib={}=lapacke", kind);
-        println!("cargo:rustc-link-lib={}=tmglib", kind);
+        if tmg {
```
Comment (IvanUkhov) [inline build.rs:42]: I suppose `LAPACKE_WITH_TMG` should be disabled too so that it’s not even compiled.

**Review by IvanUkhov — CHANGES_REQUESTED** [review]
Thanks! Please have a look at my comments.

File: build.rs (hunk 1 above)
Comment (IvanUkhov) [reply build.rs:42]: Note also that the features are listed in the readme and in the top-level doc comment. Those lists should be updated as well.

Comment (Rufflewind) [reply build.rs:42]: Updated.

**Comment by IvanUkhov** [comment]
Thank you!

**Comment by IvanUkhov** [comment]
Published.

**Pull request merged**